templates/repo/issue/comment_tab.tmpl

```diff
@@ -2,15 +2,15 @@
 	<input type="hidden" name="template-file" value="{{.TemplateFile}}">
 	{{range .Fields}}
 		{{if eq .Type "input"}}
-			{{template "repo/issue/fields/input" .}}
+			{{template "repo/issue/fields/input" Dict "ctx" $.Context "item" .}}
```

Comment:
I just recalled a previous  review
* https://github.com/go-gitea/gitea/pull/22962#discussion_r1110730669

If the  `ctx` sometimes is a `map`,  sometimes is a `Context`, that's quite misleading, and would cause more bugs in the future.

Reply:
How about renaming these `"ctx"` (in tmpl) to `"Context"` to avoid misleading?

Then there will be only:

* `{{RenderMarkdownToHtml .Context ...}}`
* `{{RenderMarkdownToHtml $.Context ...}}`

instead of `{{RenderMarkdownToHtml .ctx ...}}`, because in other templates, the "ctx" sometimes is a map

Reply:
Thanks for your suggestion. I have renamed `ctx` to `Context`, please review again.

Reply:
Thank you, I have no more question.